src/components/TemplateLibrary/index.js

```diff
@@ -36,7 +36,7 @@ const LibraryComponent = () => {
       const template = await Template.fromArchive(fileUploaded);
       const templateIdentifier = template.getIdentifier();
```

Comment:
If the issue I mentioned [here](https://github.com/accordproject/cicero-word-add-in/pull/36#issuecomment-812546885) is only with me,
then you may remove `const templateIdentifier = template.getIdentifier();` as the linter test won't pass

Reply:
Oh ya, I totally forgot about that. I'll create a new PR in sometime...